app/javascript/components/import-image-form/import-image-form.schema.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { componentTypes, validatorTypes } from '@@ddf';
+
+
+const createSchema = (providers, images, storages, buckets, diskTypes, state, setState) => ({
+  fields: [
+    {
+      component: componentTypes.SELECT,
+      initializeOnMount: true,
+      clearOnUnmount: true,
+      name: 'src_provider_id',
+      id: 'src_provider_id',
+      label: 'Choose source provider',
```

Comment:
@iv1111 labels need to be translated, Just do __('Choose source provider') or you can refer any other form in ui-classic for examples.

Reply:
done